.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Let's remove this file... should put `.vscode` in `.gitignore` to keep this from being added accidentally